### ISSUE TITLE
Move away from linux bridges and integrate openvswitch instead

### ISSUE THREAD

**Issue by muhamadazmy** [zero-os/0-core#690]
According to @delandtj linux bridges are obsolete and we should use OVS for the basic networking we do.

This means OVS should be integrated and autostarted on  zos by default before we can start creating containers and/or VMs

The reasons are:
- Performance 
- Better and more efficient port forwarding

**Comment by maxux** [comment]
I think we should include openvswitch to the initramfs, right ?

**Comment by grimpy** [comment]
@maxux yes otherwise core-0 has chicken eggs problems in it's container management.
